crates/module_system/collections/src/store_client.rs

```diff
@@ -0,0 +1,56 @@
+use ixc_core::low_level::create_packet;
+use ixc_core::result::ClientResult;
+use ixc_core::Context;
+use ixc_core_macros::message_selector;
+use ixc_message_api::code::ErrorCode;
+use ixc_message_api::header::MessageSelector;
+use ixc_message_api::AccountID;
+
+const STATE_ACCOUNT: AccountID = AccountID::new(2);
+
+const GET_SELECTOR: MessageSelector = message_selector!("ixc.store.v1.get");
+const SET_SELECTOR: MessageSelector = message_selector!("ixc.store.v1.set");
```

Comment:
i hink we should decide where to put these... 🤔 
these will be needed by the StateHandler impl AFAICT

Reply:
core?